src/Microsoft.Azure.ServiceBus/Message.cs

```diff
@@ -66,10 +72,14 @@ public string MessageId
             }
         }
 
-        /// <summary>Gets or sets a partition key for sending a transactional message to a queue or topic that is not session-aware.</summary>
-        /// <value>The partition key for sending a transactional message.</value>
-        /// <remarks>Transactions are not currently supported with this library. Messages with same partitionKey are sent to the same partition.
-        /// Max PartitionKey size is 128 chars.</remarks>
+        /// <summary>Gets or sets a partition key for sending a message to a partitioned entity.</summary>
+        /// <value>The partition key. Maximum length is 128 characters.</value>
```

Comment:
[Documentation on partitioning](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-partitioning) doesn't contain the max length info. Should it?

Reply:
We don't need all info in all places.

Reply:
I'd almost agree. Except [PartitionKey API documentation](https://docs.microsoft.com/en-us/dotnet/api/microsoft.servicebus.messaging.brokeredmessage.partitionkey?view=azureservicebus-4.1.1#Microsoft_ServiceBus_Messaging_BrokeredMessage_PartitionKey), like other API docs, should probably have that info.